tachyon/zk/plonk/examples/circuit_test_data.h

```diff
@@ -0,0 +1,92 @@
+  constexpr static std::string_view kLLast[0] = {};
+  constexpr static std::string_view kLActiveRow[0] = {};
+  constexpr static std::string_view kFixedColumns[0][0] = {};
+  constexpr static std::string_view kFixedPolys[0][0] = {};
+  constexpr static std::string_view kPermutationsColumns[0][0] = {};
+  constexpr static std::string_view kPermutationsPolys[0][0] = {};
+  constexpr static uint8_t kProof[0] = {};
+  constexpr static
+      typename CircuitTest<PCS, LS>::Point kAdviceCommitments[0][0] = {};
+  constexpr static std::string_view kChallenges[0] = {};
+  constexpr static std::string_view kTheta = "";
+  constexpr static typename CircuitTest<PCS, LS>::Point
+      kPermutationProductCommitmentsInputPoints[0][0] = {};
+  constexpr static typename CircuitTest<PCS, LS>::Point
+      kPermutationProductCommitmentsTablePoints[0][0] = {};
+  constexpr static std::string_view kBeta = "";
+  constexpr static std::string_view kGamma = "";
+  constexpr static typename CircuitTest<PCS, LS>::Point
+      kPermutationProductCommitments[0][0] = {};
+  constexpr static
+      typename CircuitTest<PCS, LS>::Point kLookupProductCommitments[0][0] = {};
+  constexpr static std::string_view kY = "";
+  constexpr static
+      typename CircuitTest<PCS, LS>::Point kVanishingHPolyCommitments[0] = {};
+  constexpr static std::string_view kX = "";
+  constexpr static std::string_view kAdviceEvals[0][0] = {};
+  constexpr static std::string_view kFixedEvals[0] = {};
+  constexpr static std::string_view kCommonPermutationEvals[0] = {};
+  constexpr static std::string_view kPermutationProductEvals[0][0] = {};
+  constexpr static std::string_view kPermutationProductNextEvals[0][0] = {};
+  constexpr static std::string_view kPermutationProductLastEvals[0][0] = {};
+  constexpr static std::string_view kLookupProductEvals[0][0] = {};
+  constexpr static std::string_view kLookupProductNextEvals[0][0] = {};
+  constexpr static std::string_view kLookupPermutedInputEvals[0][0] = {};
+  constexpr static std::string_view kLookupPermutedInputPrevEvals[0][0] = {};
+  constexpr static std::string_view kLookupPermutedTableEvals[0][0] = {};
+  constexpr static std::string_view kHEval = "";
+
+  constexpr static
+      typename CircuitTest<PCS, LS>::Point kVanishingRandomPolyCommitment{
+          "0x0000000000000000000000000000000000000000000000000000000000000001",
+          "0x0000000000000000000000000000000000000000000000000000000000000002"};
+
+  constexpr static std::string_view kVanishingRandomEval =
+      "0x0000000000000000000000000000000000000000000000000000000000000001";
+
+  static Circuit GetCircuit() { return Circuit(); }
+
+  static std::vector<Circuit> GetCircuits() {
+    Circuit circuit = GetCircuit();
+    return {circuit, std::move(circuit)};
+  }
+
+  static std::vector<typename PCS::RationalEvals> GetFixedColumns() {
+    return {};
+  }
+
+  static std::vector<typename PCS::Evals> GetInstanceColumns() { return {}; }
```

Comment:
Please remove these, if you intend to add those methods in the child class!

Reply:
The circuits should have these default methods in order to satisfy the ExampleCircuitTest typed tests. Some special circuits implement these functions differently, but others just need the empty default result (aka shuffle circuit needs the default `GetFixedColumns()` and `GetInstanceColumns()`)

Reply:
Then did you also considering implementing these as `virtual` methods?

Reply:
I did think of using `virtual`, but if I want to use virtual methods, I need to create an object of the class in order to access the methods instead of accessing them like `... ::Function()`. Is there a more elegant way to do this that you can think of?